src/style.css

```diff
@@ -1,3 +1,68 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+:root {
+  --colour: #00acdc;
```

Comment:
Variables rock 🤘 

But please fix your variable name to make it informative
Better choices are: 'active-color', 'active-blue' even 'moyo-blue'